src/lib/ReactCore_Internal.re

```diff
@@ -76,9 +78,8 @@ module Make = (Implementation: HostImplementation) => {
     | Nested(string, list(reactElement))
   and nativeElement = {
     make: int => Implementation.hostView,
```

Comment:
`make` should be `unit => Implementation.hostView` and there should be `Implementation.memoizeInstance: (int, Implementation.hostView) => unit`.